### PR TITLE
Added Item, Corrected Price And Moved Some Stuff

### PR DESCRIPTION
**Pull request by CashlessPath5**
Added [ Wither Rose, Glass Bottle, Soul Soil, Glistering Melon, Shield, etc  ] / Corrected Price [ Redstone Block, Blaze Rod] / Corrected Some rows because there were some gaps in the shop

### CODE REVIEW

File: GUIShop/shops.yml [hunk 1]
```diff
@@ -338,6 +340,11 @@ FarmingDrops:
     type: SHOP
     id: SHULKER_SHELL
     buy-price: 2000.0
+  '36':
+    type: SHOP
+    id: WITHER_ROSE
+    buy-price: 100.0
```
Comment (GerdonAbbink) [inline GUIShop/shops.yml]: Maybe a little too cheap, you have to do a lot to get them.

File: GUIShop/shops.yml [hunk 2]
```diff
@@ -489,6 +504,10 @@ Gear:
     id: SADDLE
     buy-price: 75.0
     sell-price: 30.0
+  '27':
+    type: SHOP
+    id: SHIELD
+    buy-price: 100.0
```
Comment (GerdonAbbink) [inline GUIShop/shops.yml]: Maybe reduce the price to 50, this way people can buy one just after they've joined

File: GUIShop/shops.yml (hunk 1 above)
Comment (CashlessPath5) [reply GUIShop/shops.yml]: what about 150? if u make a farm u can get 2 double chests of wither rose in 1 hour

File: GUIShop/shops.yml (hunk 2 above)
Comment (CashlessPath5) [reply GUIShop/shops.yml]: 50 is ok

**Pull request closed without merging**